### DISCUSSION TITLE
Scrobbling

### DISCUSSION THREAD

**Discussion by FazzaDazza** [Taxel/PlexTraktSync#951] [Support] [answered]
Hi there

I am using the windows install method. I have run the setup.bat and configured everything and have done a sync. I have a large library and this takes quite some time (13 hours), even after the first sync.

I would prefer to use the scrobble feature and then run the sync from time to time if things are missed. Currently I just run the plextraktsync.bat. Is there a separate file to run the scrobble or is that also included in the plextraktsync.bat ? If so, should I change the sync part in config.yml to false and leave the watch settings as is and then run the plextraktsync.bat ?

**Comment by glensc** [comment, marked as answer]
`plextraktsync.bat watch` should work:
- https://github.com/Taxel/PlexTraktSync/blob/ae900dbf58c7d79dbbe9eb27198bc70cee0ea96d/plextraktsync.bat#L19
